### ISSUE TITLE
NTR: MR Spectroscopy: guanidoacetate high in brain (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8888]
New term request MR Spectroscopy: guanidoacetate high in brain (segal_071031160504): MRS GAA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000484
- label: Reduced brain guanidinoacetate level by MRS
- parents: [Abnormal metabolic brain imaging by MRS (HP:0012705)](https://hpo.jax.org/app/browse/term/HP:0012705)
- pmid: PMID:9386672
- def: A decrease in the level of guanidinoacetate in the brain identified by magnetic resonance spectroscopy (MRS).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8888
- synonyms:
